src-plugins/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp

```diff
@@ -143,12 +141,18 @@ void medVtkViewItkDataImage4DInteractor::setInputData(medAbstractData *data)
               AppendImageSequence<float>(data,d->view,d->sequence, layer)          ||
               AppendImageSequence<double>(data,d->view,d->sequence, layer))
         {
-            d->imageData->addMetaData("SequenceDuration", QString::number(d->sequence->GetMaxTime()));
```

Comment:
there was a bug here. MaxTime was the last time of the dynamics. However, it does not take into account the duration of the last dynamic.